pkg/util/metrics_helper.go

```diff
@@ -31,6 +31,13 @@ func (m singleValueWithLabelsMap) aggregateFn(labelsKey string, labelValues []st
 	m[labelsKey] = r
 }
 
+func (m singleValueWithLabelsMap) appendUserLabelValue(user string) {
```

Comment:
Technically `prependUserLabelValue` would be a more accurate name, cause we add at the beginning.

Reply:
👍